source/dsl/BitBucketCloudDSL.ts

```diff
@@ -0,0 +1,164 @@
+import { RepoMetaData as BitBucketServerRepoMetaData } from "./BitBucketServerDSL"
+
+export interface BitBucketCloudJSONDSL {
```

Comment:
I wonder how different are the `BitBucketCloud` and `BitBucketServer` DSLs and if makes sense to have just one for both.
At a first look they seem similar to me.
My opinion about that would be that is better have an easier model for the users of this DSL (both `Danger users and the other Danger version), even if it eventually requires a major version bump.
What do you think?

Reply:
We ended up having to do this split for Danger ruby too - not sure the reasoning but I must have agreed with it then :D

Reply:
Yeah they both look similar to me too.
I’ve tried to reuse BitBucketServer’s intefaces, but later I found that they were totally different. 🤯

For example, https://github.com/danger/danger-js/blob/master/source/dsl/BitBucketServerDSL.ts#L91
only ‘message’ and ‘parents’ were the same for both of them.